src/components/manage/Toolbar/More.jsx

```diff
@@ -209,346 +169,285 @@ class More extends Component {
+                                date: (
+                                  <FormattedDate
+                                    date={content.working_copy?.created}
+                                    format={dateOptions}
+                                  />
+                                ),
+                              },
+                            )}
+                          />,
+                          {
+                            toastId: 'workingcopyapplyinfo',
+                            autoClose: 10000,
+                          },
+                        );
+                      });
+                    }}
+                  >
+                    {intl.formatMessage(messages.applyWorkingCopy)}
+
+                    <Icon
+                      name={applySVG}
+                      size="24px"
+                      title={intl.formatMessage(messages.applyWorkingCopy)}
+                    />
+                  </button>
                 </li>
-              )}
-            </Plug>
-            {path !== '' && !config.settings.excludeLinksAndReferencesMenuItem && (
-              <Plug pluggable="toolbar-more-menu-list" id="linkstoitems">
                 <li>
-                  <Link to={`${path}/links-to-item`}>
-                    {this.props.intl.formatMessage(messages.linkstoitem)}
-                    <Icon name={rightArrowSVG} size="24px" />
-                  </Link>
+                  <button
+                    aria-label={intl.formatMessage(messages.removeWorkingCopy)}
+                    onClick={() => {
+                      dispatch(removeWorkingCopy(path)).then((response) => {
+                        history.push(
+                          flattenToAppURL(content.working_copy_of['@id']),
+                        );
+                        props.closeMenu();
+                        toast.info(
+                          <Toast
+                            info
+                            title={intl.formatMessage(
+                              messages.workingCopyRemovedTitle,
+                            )}
+                          />,
+                          {
+                            toastId: 'workingcopyremovednotice',
+                            autoClose: 10000,
+                          },
+                        );
+                      });
+                    }}
+                  >
+                    {intl.formatMessage(messages.removeWorkingCopy)}
+
+                    <Icon
+                      name={removeSVG}
+                      size="24px"
+                      color="#e40166"
+                      title={intl.formatMessage(messages.removeWorkingCopy)}
+                    />
+                  </button>
                 </li>
               </Plug>
             )}
-            <Plug pluggable="toolbar-more-menu-list" id="rules">
-              {rulesAction && (
+            {content.working_copy && !content.working_copy_of && (
```

Comment:
I'm a bit concern about these changes. The rendered jsx should not differ in any case. Please make a side by side comparison. Also note, I just rebased with master.